packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenu.tsx

```diff
@@ -18,12 +19,12 @@ const StyledDropdownMenu = styled.div<{
   border: ${({ disableBorder, theme }) =>
     disableBorder ? 'none' : `1px solid ${theme.border.color.medium}`};
   border-radius: ${({ theme }) => theme.border.radius.md};
-
   box-shadow: ${({ theme }) => theme.boxShadow.strong};
 
   display: flex;
 
-  height: 100%;
+  height: ${({ isCustomSlashMenu }) =>
+    isCustomSlashMenu ? 'fit-content' : '100%'};
```

Comment:
@Hitarthsheth07 fit-content was not working everywhere?

Reply:
It does work every, should I change it to fit-content

Reply:
If it works everywhere yes

Reply:
Ob it!

Reply:
Done!